### PR TITLE
merge dev-it to main

### PR DESCRIPTION
**Pull request by sistella**
First commit into main for italian language

### CODE REVIEW

File: content/it/infrastructure_as_code.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Infrastructure as Code (IaC - Infrastruttura come Codice)
+status: Feedback Appreciated
```
Comment (jihoon-seo) [inline content/it/infrastructure_as_code.md:3]: If this entity is okay to be exposed, then please change the `status` to `Completed`:

```suggestion
status: Completed
```

**Comment by jihoon-seo** [comment]
Preview: https://deploy-preview-735--cncfglossary.netlify.app/it/

Check list for maintainers for a PR to make localization live.
 - [ ] validate all files which are not in the localization directory (content/xx/*)
 - [ ] check i18n/xx.toml exist
 - [ ] check l10n configuration in config.toml
 - [ ] acknowledge to initial localization contributors list
 - [ ] check Preview works

**Review by jihoon-seo — COMMENTED** [review]
Dear Italian l10n team,
thank you for your devotion and hard work on this project!

---

The title of some terms are in Italian, while others are in English.
image

@fsbaraglia @meryem-ldn @annalisag-spark @sistella 
Are you okay with this as-is? Or do you want to update some of those? 😊

File: content/it/infrastructure_as_code.md (hunk 1 above)
Comment (annalisag-spark) [reply content/it/infrastructure_as_code.md:3]: @jihoon-seo thank you. Go, go, go!

**Review by annalisag-spark — APPROVED** [review]

**Comment by jihoon-seo** [comment]
Preview of one of terms: https://deploy-preview-735--cncfglossary.netlify.app/it/abstraction/

Feedback section is displayed like this:
image

@sistella Could you add some entries to [`it.toml` of `dev-it` branch](https://github.com/cncf/glossary/blob/dev-it/i18n/it.toml)?

Examples:
- https://github.com/cncf/glossary/blob/540e516abc4950f80463e3997d09355d6a671444/i18n/en.toml#L63-L71
- https://github.com/cncf/glossary/pull/665/files#diff-fd5507f0042e10570f36b7804477e5502fbb6f6a159db8e0a5e86f80c981be06
- https://github.com/cncf/glossary/pull/519/files#diff-bf91abcf6a5d6fc7b18204067500f96305db8352586d24a549a23cd973daa5a5

PTAL! 😊

File: content/it/cloud_native_tech.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Cloud Native Technology
```
Comment (annalisag-spark) [inline content/it/cloud_native_tech.md:2]:
```suggestion
title: Tecnologia Cloud Native
```

**Review by annalisag-spark — APPROVED** [review]

**Review by jihoon-seo — APPROVED** [review]
@sistella Thank you, and LGTM!

**Review by seokho-son — APPROVED** [review]
LGTM :)

Thank you! Italian l10n team !!

**Pull request merged**